show/src/lib.rs

```diff
@@ -73,8 +89,64 @@ impl Args for Options {
                 show_proj_id,
                 show_ssh_key,
                 show_profile_id,
+                urn,
             },
             vec![],
         ))
     }
 }
+
+pub fn run(mut options: Options) -> anyhow::Result<()> {
+    let profile = profile::default()?;
+    let sock = keys::ssh_auth_sock();
+    let (_, storage) = keys::storage(&profile, sock)?;
+
+    if options == Options::default() {
+        options.show_proj_id = true;
+        options.show_peer_id = true;
+        options.show_self = true;
+        options.show_profile_id = true;
+        options.show_ssh_key = true;
+    }
+
+    if let Some(urn) = options.urn {
+        use librad::git::identities::any;
+        use librad::git::identities::Error;
+        let payload = any::get(&storage, &urn)
+            .map(|o| o.map(|p| p.payload()))
+            .or_else(|_| any::get(&storage, &urn).map(|o| o.map(|p| p.payload())))
```

Comment:
Why would we get the payload a second time if the first time fails?

Reply:
right https://github.com/radicle-dev/radicle-client-tools/pull/45/files#diff-0557537217856d24931543b3a7d656da802f151f3f0d33b787818f7aaf891472R114-R115